### PR TITLE
OSDOCS-5584-RN:updates 4.13 tables

### PR DESCRIPTION
**Pull request by JoeAldinger**
[OSDOCS-5584](https://issues.redhat.com//browse/OSDOCS-5584): updates the Tech Preview and Dep. tables in 4.13 RNs

Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-5584

Link to docs preview:
https://58100--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-deprecated-features

QE review:
- [ ] QE has approved this change.

Additional information:
Includes updates for: https://issues.redhat.com/browse/MULTIARCH-3240

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -1188,9 +1193,9 @@ In the following tables, features are marked with the following statuses:
 |Feature |4.11 |4.12 |4.13
 
 |Serverless functions
-|Technology Preview
-|Technology Preview
-|Technology Preview
+|General Availability
+|General Availability
+|General Availability
```
Comment (opayne1) [inline release_notes/ocp-4-13-release-notes.adoc:1198]: Wondering if this should just be removed from the list?

**Review by opayne1 — COMMENTED** [review]
Just one question and also for the [Web Console](https://github.com/openshift/openshift-docs/pull/58100/files#diff-5a1bef860ba4f103b12cdc2db661140fa691167a8b50abea180008754482951cR1230) I just found out that the Multicluster console feature is on hold and I am removing the docs. We should probably remove that from the table as well. Dynamic Plugin was GA starting in 4.12.

**Comment by nainaz** [comment]
Serverless functions is GA so it shouldn't be in the Technology Preview section. But moved to GA section.

**Comment by SNiemann15** [comment]
@JoeAldinger The changes you pulled in from my [PR](https://github.com/openshift/openshift-docs/pull/57256) look good. 
I'll close mine. 

@manojnkumar @alishaIBM In case the Power TP entries need to be removed from the **Multi-Architecture Technology Preview tracker** table please inform Joe.

**Comment by cbremble** [comment]
LGTM re: removing Serverless TP issue mention. Thanks!

**Pull request merged**